packages/manager/apps/dedicated/client/app/account/user/users/sso/details/user-users-sso-details.controller.js

```diff
@@ -0,0 +1,20 @@
+export default /* @ngInject */ function UserAccountUsersSSODetailsCtrl(
+  $scope,
+  UseraccountUsersService,
+) {
+  function initIdentityProvider() {
+    UseraccountUsersService.getIdentityProvider()
+      .then((identityProvider) => {
+        $scope.identityProvider = identityProvider;
+      })
+      .catch(() => {
+        $scope.identityProvider = null;
+      });
+  }
+
+  function init() {
+    initIdentityProvider();
+  }
+
+  init();
+}
```

Comment:
May be we could use `class` syntax ?
```suggestion
export default class UserAccountUsersSsoDetailsCtrl {
  /* @ngInject */
  constructor($scope, UseraccountUsersService) {
     this.$scope = $scope;
     this.useraccountUsersService = UseraccountUsersService;
 }
 
 $onInit() {
    this.useraccountUsersService.getIdentityProvider()
      .then((identityProvider) => {
        $scope.identityProvider = identityProvider;
      })
      .catch(() => {
        $scope.identityProvider = null;
      });
  }
}
```

Reply:
Done